src/Transformation/Factory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Transformation;
+
+/**
+ * Factory for basic transformations.
```

Comment:
Other factories have some description what their supposed usage is. Maybe something like "For purpose and usage see README.md" would be better than duplicate description here.